lib/utils.js

```diff
@@ -194,10 +194,9 @@ async function clearSystemFiles (wda) {
     detached: true,
     stdio: ['ignore', 'pipe', 'pipe'],
   });
-  await cleanupTask.start(0);
   // Do not wait for the task to be completed, since it might take a lot of time
   // We keep it running after Appium process is killed
-  cleanupTask.proc.unref();
+  await cleanupTask.start(0, true);
```

Comment:
just a random reflection since this is an example of something in our API design i've been wanting to have a discussion about: more and more I've been thinking that the "default options object" way of passing parameters to a method is easier to read/understand than the "positional arguments" way. so this:

```js
await cleanupTask.start({startTimeout: 0, unref: true});
```
instead of what we currently have.

i don't think we need to go through and change stuff all around our codebase, just wondering what other maintainers think about adopting that pattern more going forward? i know we already do it a fair bit.

Reply:
I generally prefer that pattern, myself.